contributor/outline.md

```diff
@@ -1,64 +1,138 @@
+    - When you have technical needs that are outside of the domain of your project,
+    look around to see if there is a shared solution that you can use.
+    - Briefly enumerate the benefits of using shared solutions.
+  - Using shared solutions puts you in a position to become a _Contributor_.
+    - There will be times when you wish that the shared solution has some new feature.
+    - Don't _Wait it Out_ or _Work Around_, spend your time and energy making the change directly in the shared project.
+    - Briefly enumerate the benefits of contributing.
+    This should be taken from the **Benefits** segment.
+    - You may need to spend a bit of time explaining these benefits to your management hierarchy in order to have their support to spend your time contributing.
+    - An InnerSource _Contributor_ knows that it doesn't matter where the code goes as long as they get the needed functionality.
+  - Contributing applies to all levels of project managements, not just code.
+    - Opening issues.
+    - Improving documention.
+    - Fixing builds.
+    - Nothing too small to contribute (e.g. documentation typo).
+      - [Here is one](https://github.com/tensorflow/models/pull/4784) that I made to `tensorflow/models`.
+      A simple label change in a graph.
 
-## Wrap-up/Final Thoughts
-   - Treat contribution as a means to help enhance your knowledge (and career)
-   - Remember you don’t necessarily have all of the answers
-   - Release early, release often (‘Perfect is the enemy of good enough’)
-   - Have fun!
+## The Contributor Ethos
 
+  - Introduce the guest in a home analogy.
+  A good guest:
+    - Knocks at the door.
+    - Anticipates and follows house rules.
+    - Understands they are not the home owner and acts accordingly.
+  - Let's talk about how each of these ideas applies in InnerSource.
+  - Getting in
+    - In the real world their are several ways that guests are invited in.
+      - An "open" sign lets them know they can walk right in without knocking.
+      - Knock on the door unexpected.
+      - Knock on the door only when invited.
+    - With InnerSource projects there are several ways that code gets in.
+      - Sometimes you just open a pull request.
+      - At times you may need to get a feature request accepted before opening the pull request that fulfills it.
+      - You may need to talk to the host team in person before interacting with the project digitally.
+      In the same way that real-life hosts don't always appreciate a stranger knocking on their door, some InnerSource hosts appreaciate some heads-up before seeing a pull request get opened to their repo.
+      - In all of these interactions, be prepared to "sell" your contribution to the host team.
+        - Articulate the value that the contribution will bring to their ecosystem.
+        - Remind them that they are getting free engineering help with their project.
+        - Offer to fulfill a [30-day warranty] on your submission.
```

Comment:
Could hint here that this alleviates the fear-of-maintenance problem that might arise in the host team. (Advertisement for managers / host teams watching this segment) (Effectively a glimpse into one of the motivations for the pattern)